test/unit/graphql_test.lua

```diff
@@ -1057,3 +1058,10 @@ function g.test_boolean_coerce()
     t.assert_error_msg_contains('Could not coerce value "value" with type "string" to type boolean',
             validate, test_schema, parse([[ { test_boolean(value: "value") } ]]))
 end
+
+function g.test_util_map_name()
+    local res = util.map_name(nil, nil)
+    t.assert_equals(res, {})
+    res = util.map_name({ { name = 'a' }, { name = 'b' }, }, function(v) return v end)
```

Comment:
I think we can test tables that contains not only numeric keys since we use "pairs" inside this function.

Reply:
Added new test case